test/test_runner.sh

```diff
@@ -26,7 +26,7 @@ function execute() {
 }
 
 # Tests to run. Defaults to all.
-TESTS=${*:-.}
+TESTS=${TESTS:-.}
```

Comment:
This seems to be the cause of the error on ci/kata-jenkins : rather than running only the .bats file(s) given as parameter, the script is now expecting an environment variable to be set. Since our test script is not setting this variable, we default to running all bats files.

I can understand the change, but I'll need to adapt our test script. In the meantime, ci/kata-jenkins will fail systematically with this commit. I don't know if there are other jobs relying on it?

Reply:
honestly I didn't realize how this worked and thought it was just broken lol, I've reverted the change, thanks for the insight